packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -70,7 +70,7 @@ class BottomSheet extends StatefulWidget {
   /// [showModalBottomSheet], for modal bottom sheets.
   const BottomSheet({
     Key? key,
-    this.animationController,
+    required this.animationController,
```

Comment:
Studying this a little further, it seems the animationController is only required if enableDrag is true.